lib/core/public/cleanup-plugins.js

```diff
@@ -23,7 +23,7 @@ function cleanupPlugins(resolve, reject) {
     });
   });
 
-  axe.utils.toArray(document.querySelectorAll('frame, iframe')).forEach(function (frame) {
+  axe.utils.getFrames().forEach(function (frame) {

```

Comment:
You should be able to use `axe.utils.querySelectorAll(axe._tree, 'frame, iframe')` instead.